meta-builder/PROJECT

```diff
@@ -11,3 +11,15 @@ resources:
 - group: synopsys
   version: v1
   kind: OpsSight
+- group: synopsys
+  version: v1
+  kind: Polaris
+- group: synopsys
+  version: v1
+  kind: Reporting
+- group: synopsys
+  version: v1
+  kind: PolarisInfra
```

Comment:
What I heard from Viral is that, we don't want to maintain infra as a separate CRD.

And also, why Polaris DB is separate entity? Why it is not merged with Polaris?

Reply:
The reason i kept it independent of Polaris is all the database services will be moving away from core k8s objects (Ship coming in). So it would be easier for us to remove CRD directly rather making changes.

Reply:
Regarding Infra, I added it initially. But kubebuilder is not favourable to remove them through a command.

I'll do it manually.